android/capacitor/src/main/java/com/getcapacitor/plugin/LocalNotification.java

```diff
@@ -0,0 +1,199 @@
+package com.getcapacitor.plugin;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.app.NotificationManagerCompat;
+
+import com.getcapacitor.JSArray;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.NativePlugin;
+import com.getcapacitor.Plugin;
+import com.getcapacitor.PluginCall;
+import com.getcapacitor.PluginMethod;
+import com.getcapacitor.plugin.notification.Notification;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Plugin for creating local notification
+ */
+@NativePlugin()
+public class LocalNotification extends Plugin {
```

Comment:
Class should be LocalNotifications, as that's the iOS name, otherwise it won't work with current API as the plugin execution is based on the class name.